boards/nrf6310/Makefile.include

```diff
@@ -19,3 +18,9 @@ export RESET_FLAGS = $(BINDIR)
 
 # setup serial terminal
 include $(RIOTMAKE)/tools/serial.inc.mk
+
+# use jlink to program this board
+PROGRAMMER = jlink
```

Comment:
does it make sense to use `?=` here? I think at least in theory it should be also possible to use openocd to program this device, right?!

Reply:
> I think at least in theory it should be also possible to use openocd to program this device, right?!

Indeed, I'll make the change. I also think in adding the possibility to use a variable containing the list of supported programmers, like `SUPPORTED_PROGRAMMERS`. The common makefile.include would check if the supplied programmer is there and return an error if it's not the case. What do you think ?